modules/combined/test/tests/mechanical_contact_constraint/blocks_2d/frictionless_kinematic_gap_offsets.i

```diff
@@ -0,0 +1,202 @@
+# this test is the same as frictionless_kinematic test but designed to test the gap offset capability
+# gap offsets with value of 0.01 were introduced to both master and slave sides in the initial mesh
+# these values were accounted using the gap offset capability to produce the same result as if no gap offsets were introduced
+[Mesh]
+  file = blocks_2d_gap_offset.e
+[]
+
+[GlobalParams]
+  displacements = 'disp_x disp_y'
+[]
+
+[Variables]
+  [./disp_x]
+  [../]
+  [./disp_y]
+  [../]
+[]
+
+[AuxVariables]
+  [./penetration]
+  [../]
+  [./inc_slip_x]
+  [../]
+  [./inc_slip_y]
+  [../]
+  [./accum_slip_x]
+  [../]
+  [./accum_slip_y]
+  [../]
+  [./master_gap_offset]
+  [../]
+  [./slave_gap_offset]
+  [../]
+  [./mapped_master_gap_offset]
+  [../]
+[]
+
+[Functions]
+  [./vertical_movement]
+    type = ParsedFunction
+    value = -t
+  [../]
+[]
+
+[SolidMechanics]
```

Comment:
This needs to use TM

Reply:
@recuero has some pull requests in right now that convert some of these tests to TM and fix some issues with SM.  We should probably just wait until those go in.

Reply:
> This needs to use TM

This test was updated to use TM.